build/webpack.config.js

```diff
@@ -36,19 +36,18 @@ export const webpackHotMiddlewareEntry = `webpack-hot-middleware/client?${_.map(
   quiet: false,           // Set to true to disable all console logging.
 }, (val, key) => `&${key}=${val}`).join('')}`
 
-const APP_ENTRY = [
-  paths.docsSrc('index.js'),
-]
+const APP_ENTRY = paths.docsSrc('index.js')
 
 webpackConfig.entry = __DEV__ ? {
   app: [
     webpackHotMiddlewareEntry,
-    ...APP_ENTRY,
+    APP_ENTRY,
   ],
   vendor: [
     webpackHotMiddlewareEntry,
-    ...config.compiler_vendor,
-  ],
+  ].concat(
+    config.compiler_vendor
+  ),
```

Comment:
`webpackHotMiddlewareEntry` is a string and `config.compiler_vendor` is an array.